lib/tableau/router.ex

```diff
@@ -6,6 +6,8 @@ defmodule Tableau.Router do
 
   require Logger
 
+  @base_path Path.join("/", Application.compile_env(:tableau, :config)[:base_path] || "")
```

Comment:
I don't think you actually need the `|| ""` since it defaults to `""`

Reply:
In my testing it defaults to nil, which causes `Path.join/2` to raise an error.

Reply:
Oh, sorry, I was thinking of the default coming from the struct. This is fine then, should also add to the other spot instead of that case expression

Reply:
I took a closer look at this.  Turns out there is a small difference in the arguments accepted by `Application.get_env` and `Application.compile_env`.  I believe the forms I used are correct, and tested it with and without a `base_path` configuration.